tasks/src/test/java/org/example/aSerejaAndDima/ASerejaAndDima2Test.java

```diff
@@ -0,0 +1,94 @@
+package org.example.aSerejaAndDima;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+class ASerejaAndDima2Test {
+    private final ASerejaAndDima2.ASerejaAndDima2Verifier aSerejaAndDima2Verifier = new ASerejaAndDima2.ASerejaAndDima2Verifier();
+    private final ASerejaAndDima2 aSerejaAndDima2 = new ASerejaAndDima2();
+    private static List<Integer> listOfCardsInsert;
+
+    @BeforeAll
+    static void insertListOfCards() {
+        listOfCardsInsert = List.of(1, 2, 3, 4, 5, 6, 7);
+    }
+
+    @Test
+    void shouldReturnTrueWhenNumberOfCardsFrom1To1000() {
+        //given - arrange
+        int numberOfCards = 1;
+
+        //when - act
+        boolean checkNumberOfCards = aSerejaAndDima2Verifier.numberOfCardsVerifier(numberOfCards);
+
+        //then - assert
+        Assertions.assertTrue(checkNumberOfCards);
+    }
+
+    @Test
+    void shouldReturnTrueWhenListOfCardsFrom1To1000AndNotNull() {
+        //given - arrange
+        List<Integer> listOfCards = List.of(4, 1, 2, 10);
+
+        //when - act
+        boolean checkListOfCards = aSerejaAndDima2Verifier.validNumbersInListOfCardsVerifier(listOfCards);
+
+        //then - assert
+        Assertions.assertTrue(checkListOfCards);
+    }
+
+    @Test
+    void shouldReturnFalseWhenNumberOfCardsAndListOfCardsHaveDifferentSize() {
+        //given - arrange
+        int numberOfCards = 3;
+        List<Integer> listOfCards = List.of(4, 1, 2, 10);
+
+        //when - act
+        boolean check = aSerejaAndDima2Verifier.inputSizeOfCardsVerifier(numberOfCards, listOfCards);
+
+        //then - assert
+        Assertions.assertFalse(check);
+
+    }
+
+    @Test
+    void shouldReturnTrueWhenNumbersInListOfCardsAreDistinct() {
+        //given
+        //List<Integer> listOfCards = List.of(1, 2, 3, 4, 5, 6, 7);
+        //Czy może nie być 'given', jeżeli generujemy dane poprzez odrębną metodę statyczną z @BeforeAll??
```

Comment:
dla czytelnosci zazwyczja zostawiam given i robie np List<Integer> list = generateList();
Tez zastanawiam sie nad uzytecznoscia tej metody. Nie chce mi sie liczyc w ilu miejscach jej uzywasz ale jezeli np w 2 i jest to tak banalny setup to mozna to olac. Z reszta pisalem wyzej ze lepiej zrobic z tego zmienna po prostu w klasie